src/routes/api/users.js

```diff
@@ -4,25 +4,25 @@ const router = express.Router();
 
 router.get("/user", (req, res, next) => {
   res.status(200).json({
-    message: "successfully sent query"
+    message: res.__("successfully sent query")
   });
```

Comment:
Why are you adding 2 underscore ?

Reply:
Can you please do rebase first?

Reply:
@Duncanwar for the message to be translated.

Reply:
@BiGelio Done.